### PR TITLE
smoke tests for kyma-dashboard stage/prod

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: templates/data/kyma-dashboard-smoke-test-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              preset-sa-vm-kyma-integration: "true"
+              preset-gc-project-env: "true"
+        jobConfigs:
+          - repoName: kyma-project/kyma-dashboard
+            jobs:
+              # presubmits
+              - jobConfig:
+                  name: "pre-kyma-dashboard-smoke-test-prod"
+                  optional: false
+                  run_if_changed: "^Dockerfile.*|^package.json|^environments/prod/|^extensions|^gulpfile.js|^Makefile"
```
Comment (akucharska) [inline templates/data/kyma-dashboard-smoke-test-prod.yaml:27]: can we add smoke test files here?

File: templates/data/kyma-dashboard-smoke-test-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                  local:
+                    - "jobConfig_default"
+                    - "request_small"
+                    - "vm_job_template_k3s"
+                    - "vm_job_labels_template"
+              # postsubmits
+              - jobConfig:
+                  name: "post-kyma-dashboard-smoke-test-prod"
+                  optional: false
+                  run_if_changed: "^Dockerfile.*|^package.json|^environments/prod/|^extensions|^gulpfile.js|^Makefile"
```
Comment (akucharska) [inline templates/data/kyma-dashboard-smoke-test-prod.yaml:48]: can we add smoke test files here?

File: templates/data/generic_component_kyma_dashboard_data.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ templates:
                   args:
                     - "."
                     - "release-dev"
-                  run_if_changed: "^Dockerfile.*|^package.json|^environments/dev/|^extensions|^gulpfile.js|^Makefile"
+                  run_if_changed: "^Dockerfile.*|^package.json|^environments/dev/|^extensions|^gulpfile.js|^Makefile|^tests/"
```
Comment (akucharska) [inline templates/data/generic_component_kyma_dashboard_data.yaml:13]: can we add smoke test files for stage and dev jobs?

**Review by akucharska — CHANGES_REQUESTED** [review]

File: templates/data/generic_component_kyma_dashboard_data.yaml (hunk 3 above)
Comment (mrCherry97) [reply templates/data/generic_component_kyma_dashboard_data.yaml:13]: for dev we are running whole integration tests, so smoke tests are unnecessary. for stage adjusted

**Review by akucharska — APPROVED** [review]

**Review by halamix2 — APPROVED** [review]

**Pull request merged**